### PR TITLE
Converted newly added samples in java

### PR DESCRIPTION
**Pull request by anudeepsharma**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@anudeepsharma__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
> using System.Reflection;

hmmm... something is missing at the top of the file ;) 

---
Refers to: Samples/ResourceManagement/Compute/ManageManagedDisks/Properties/AssemblyInfo.cs:1 in c290178. [](commit_id = c290178fcee3e89e65b663fc50330af5245e9016, deletion_comment = False)

**Comment by hovsepm** [comment]
> using System.Reflection;

Why did you remove License header here?

---
Refers to: Samples/ResourceManagement/Compute/ManageVirtualMachine/Properties/AssemblyInfo.cs:1 in c290178. [](commit_id = c290178fcee3e89e65b663fc50330af5245e9016, deletion_comment = False)

File: Samples/ResourceManagement/Compute/ManageVirtualMachineScaleSet/Program.cs [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
+// Licensed under the MIT License. See License.txtinthe project root for license information.
```
Comment (hovsepm) [inline Samples/ResourceManagement/Compute/ManageVirtualMachineScaleSet/Program.cs]: ? License.txtinthe looks wrong ;)

**Comment by hovsepm** [comment]
> using System.Reflection;

Please bring back the license header. 

---
Refers to: Samples/ResourceManagement/Compute/ManageVirtualMachineScaleSet/Properties/AssemblyInfo.cs:1 in c290178. [](commit_id = c290178fcee3e89e65b663fc50330af5245e9016, deletion_comment = False)

**Comment by anudeepsharma** [comment]
Fixed comments

**Pull request merged**